pkg/controller/knativeeventingkafka/knativeeventingkafka_controller.go

```diff
@@ -23,6 +24,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+const (
+	ns = "knative-eventing"
```

Comment:
Sharing a ns with another component _may_ be a problem. But if it is, I'd prefer to fix manifestival than force you to use your own ns. Just be aware that if an operator creates a ns, it will probably try to delete that ns when the associated CR is deleted. If the ns was already there, it won't.

Reply:
This latest iteration of the Kafka operator places all the ns-scoped k8s objects it creates into the knative-eventing namespace, which the code assumes to exist, since the Kafka objects depend on Eventing being installed [in the knative-eventing namespace]. So I think we're in the clear wrt to the Kafka operator trying to remove the knative-eventing namespace.